test/test_kubeclient.rb

```diff
@@ -106,11 +106,11 @@ def test_get_all
     assert_instance_of(EntityList, result['pod'])
```

Comment:
Does it make sense to add also Kubeclient:: before EntityList now? Its scope should also be limited to Kubeclient, but the only difference is that it was not created dynamically